pkgs/development/libraries/science/math/cudnn/extension.nix

```diff
@@ -111,6 +111,7 @@ final: prev: let
     "11.4" = "8.3.2";
     "11.5" = "8.3.2";
     "11.6" = "8.3.2";
+    "11.7" = "8.3.2";
```

Comment:
we should get around to upgrading cuDNN to 8.4.0 at some point, but that's not necessary for this PR...

Reply:
It has been updated since I've opened this PR.
I've updated it.